src/category_theory/abelian/exact.lean

```diff
@@ -61,4 +61,18 @@ begin
     simp [h.2] }
 end
 
+/-- If `(f, g)` is exact, then `image.ι f` is a kernel of `g`. -/
+def is_limit_image [h : exact f g] : is_limit (kernel_fork.of_ι (image.ι f)
+  ((epi_iff_cancel_zero (factor_thru_image f)).1 (by apply_instance) _ (image.ι f ≫ g)
+  (by simp [h.1]))) :=
```

Comment:
Does it make sense to have an auxiliary definition for the argument of `is_limit _` here? It's... quite a mouthful!